accepted/0000-workspaces-command.md

```diff
@@ -0,0 +1,31 @@
+- Start Date: 2017-05-04
+- RFC PR:
+
+# Summary
+
+This document specify a new command, that can be used to execute subcommands inside a project workspaces.
+
+# Motivation
+
+With the addition of the Workspace feature, it will become handy to be able to execute commands inside other workspaces than the one in the current directory.
+
+# Detailed design
+
+This RFC suggests to add the two following commands:
+
+```
+$> yarn workspace <project-name> pwd
+$> yarn workspace <project-name> <command-name> ...
```

Comment:
> We've been using lerna run and lerna exec for a little while. I haven't heard of anyone confused by it. I think we can mitigate that with documentation if it does become an issue.

Fine by me, then.

> The other thing that we can do to make this useful cross-platform is to specify the $PATH the way package.json#scripts do.

Ooooh, I like it, I often end up writing `"env": "env"` in my scripts just because I want to see the env my scripts will use. That could be an interesting use case for `yarn exec` (no workspace).

Ok, I'll update the document to add this.